accounting/create_db.py

```diff
@@ -10,50 +10,45 @@
 # SPDX-License-Identifier: LGPL-3.0
 ###############################################################
 import sqlite3
+import pandas as pd
+import logging
 
 
-def create_db():
+LOGGER = logging.basicConfig(filename="accounting/db_creation.log", level=logging.INFO)
```

Comment:
When I didn't have the `accounting` subdirectory, logging failed. Either recover from such failure or dump the log file into the current working directory? Ultimately you may want to make this configurable but the project is so early I gather you want to make fastest progress with core business logic.